engine/crates/engine-v2/config/src/v2.rs

```diff
@@ -14,6 +15,10 @@ pub struct Config {
 
     /// Additional configuration for our subgraphs
     pub subgraph_configs: BTreeMap<SubgraphId, SubgraphConfig>,
+
+    /// Caching configuration
+    #[serde(default)]
+    pub cache_config: BTreeMap<CacheConfigTarget, CacheConfig>,
```

Comment:
should be a struct imho, like:
```
pub struct CacheConfig {
  rules: BTreeMap<CacheConfigTarget, CacheControl>
}
```
Would provide more flexibility in case of changes later.